extensions/3rdparty/tabber/tabber.php

```diff
@@ -2,26 +2,26 @@
 
 # Credits
 $wgExtensionCredits['parserhook'][] = array(
-    'name'=>'Tabber',
-    'author'=>'Eric Fortin',
-    'url'=>'http://www.mediawiki.org/wiki/Extension:Tabber',
-    'description'=>'Create tabs that contain wiki compatible based data',
-    'version'=>'1.01'
+    'name' => 'Tabber',
+    'author' => 'Eric Fortin',
+    'url' => 'http://www.mediawiki.org/wiki/Extension:Tabber',
+    'description' => 'Create tabs that contain wiki compatible based data',
+    'version' => '1.01'
 );
 
-//Avoid unstubbing $wgParser on setHook() too early on modern (1.12+) MW versions, as per r35980
+// Avoid unstubbing $wgParser on setHook() too early on modern (1.12+) MW versions, as per r35980
 $wgHooks['ParserFirstCallInit'][] = 'wfTabber';
 
 /**
  * @param Parser $parser
  * @return bool
  */
-function wfTabber(&$parser) {
+function wfTabber( Parser $parser ) {
 	$parser->setHook( 'tabber', 'renderTabber' );
 	return true;
 }
 
-function renderTabber( $paramstring, $params, $parser ){
+function renderTabber( $paramstring, $params, $parser ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$params". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)